pyodbc.cp311-win_amd64.pyd,sha256=poW0tN2ae1sh0XWqfVwM96WgSlBx17O830pazPsgJ5I,124928
pyodbc-5.0.0a2.data/data/pyodbc.pyi,sha256=QTp2pfmOBLL05hwIemSQacwgB96OC-N0bM0rm9fAxsw,33324
pyodbc-5.0.0a2.dist-info/LICENSE.txt,sha256=mFnIV3DJG3hmsPdpyTjjbWTQJgFBtSm5R7Tq2WiD6rQ,868
pyodbc-5.0.0a2.dist-info/METADATA,sha256=gMye26yUs0mRTD9SeAUDR--haOXzd2J2OS2RCLxNuUw,2485
pyodbc-5.0.0a2.dist-info/WHEEL,sha256=badvNS-y9fEq0X-qzdZYvql_JFjI7Xfw-wR8FsjoK0I,102
pyodbc-5.0.0a2.dist-info/top_level.txt,sha256=30jvYcCul6_wA26OT5c068GAHzqQLpd9WfQ0mSSsDTU,7
pyodbc-5.0.0a2.dist-info/RECORD,,
